### PR TITLE
Added badges to match flip-remote project

### PR DESCRIPTION
**Pull request by paigeribera**
Added all the badges that were on the flip-remote project. Issue #5

### CODE REVIEW

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Looks good.
We don't want to delete package.json file.
Can you add that file back.

**Comment by paigeribera** [comment]
Okay! Sorry about that. I think I've fixed it now.

File: package.json [hunk 1]
```diff
@@ -29,5 +31,9 @@
   "bugs": {
     "url": "https://github.com/arshadkazmi42/strmat/issues"
   },
-  "homepage": "https://github.com/arshadkazmi42/strmat#readme"
+  "homepage": "https://github.com/arshadkazmi42/strmat#readme",
+  "dependencies": {
```
Comment (arshadkazmi42) [inline package.json]: This does not belongs here.
Can you remove the dependencies.
It does not uses any dependencies and same package we cannot use as dependency.
So complete depencies object should be removed

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
There is a minor change and it will be good to go.

**Comment by paigeribera** [comment]
okay, I've gotten rid of those dependencies.

**Review by arshadkazmi42 — APPROVED** [review]
Awesome 💯

**Pull request merged**